packages/data-service/src/instance-detail-helper.ts

```diff
@@ -133,12 +135,19 @@ export async function getInstance(
       getParameter: 1,
       featureCompatibilityVersion: 1,
     }).catch(() => null),
-
-    runCommand(adminDb, { atlasVersion: 1 }).catch(() => {
-      return { atlasVersion: '', gitVersion: '' };
-    }),
```

Comment:
Would we not run `atlasVersion: 1` anymore? This seems to always have been one of the better Atlas identification checks (although it only works for shared-tier clusters)